api/src/main/java/me/lokka30/treasury/api/economy/account/PlayerAccount.java

```diff
@@ -108,9 +108,12 @@ default void retrievePermissions(
     ) {
         Objects.requireNonNull(player, "player");
         Objects.requireNonNull(subscription, "subscription");
-        subscription.succeed((getUniqueId().equals(player)
-                ? ALL_PERMISSIONS_MAP
-                : Collections.emptyMap()));
+
```

Comment:
useless diff, your code does the same as the other code. revert

Reply:
Error with the diff.